index.css

```diff
@@ -161,7 +166,8 @@ a {
   margin-left: -10px;
 }
 
-.work-section {
+.card {
+  box-sizing: border-box;
   display: flex;
```

Comment:
- [ ] Please used the media query for the Desktop breakpoint in your CSS.
![res](https://user-images.githubusercontent.com/55596970/167157660-9db036c1-632e-4e8f-ab47-be4c950896db.png)